mod.sp

```diff
@@ -12,4 +12,10 @@ mod "azure_compliance" {
     description = "Run individual configuration, compliance and security controls or full compliance benchmarks for CIS, HIPAA HITRUST, NIST across all your Azure subscriptions using Steampipe."
     image       = "/images/mods/turbot/azure-compliance-social-graphic.png"
   }
+
+  requires {
+    plugin "azuread" {
```

Comment:
@khushboo9024 this also requires the `azure` plugin as well? If so, can you please add it to this section as well